bodhi/server/templates/master.html

```diff
@@ -24,8 +24,10 @@
     % else:
         <title>${update.title} - Fedora Updates System</title>
     % endif
-    <link href="${request.static_url('bodhi:server/static/bootstrap/css/bootstrap.css') }" rel="stylesheet" />
-    <link href="${request.static_url('bodhi:server/static/font-awesome-4.4.0/css/font-awesome.min.css') }" rel="stylesheet" />
+    <link href="https://apps.fedoraproject.org/global/fedora-bootstrap-1.0.1/fedora-bootstrap.css" rel="stylesheet" type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css' rel='stylesheet' type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/hack.css' rel='stylesheet' type='text/css' />
+    <link href='https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/font-awesome.css' rel='stylesheet' type='text/css' />
```

Comment:
I do have the font-awesome packages installed on my system but my browser doesn't seem to have all the fonts it needs (and I assume the CORS restriction is blocking it). Quite a few things are missing, like the search icon:

![bodhi_screenshot1](https://cloud.githubusercontent.com/assets/354506/23673443/550cef14-0340-11e7-8f14-1e88cc5e5e02.png)